client/me/account-password/index.jsx

```diff
@@ -188,7 +188,7 @@ const AccountPassword = createReactClass( {
 						}
 						onClick={ this.handleSaveButtonClick }
 					>
-						{ this.state.savingPassword ? translate( 'Saving…' ) : translate( 'Save Password' ) }
+						{ this.state.savingPassword ? translate( 'Saving…' ) : translate( 'Save password' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.